Dev/GitHub/notifications.30s.py

```diff
@@ -1,30 +1,71 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# Github Notifications by Keith Cirkel
-# Fetch Github notifications in your menu bar!
+
+# <bitbar.title>GitHub Notifications</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
```

Comment:
Only one author is currently supported on https://getbitbar.com/ - should we add support for comma separated authors?